cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/connector/resource/AzureDatabaseResourceService.java

```diff
@@ -354,7 +354,22 @@ private List<CloudResource> getResources(List<CloudResource> resources, Resource
     private void deployDatabaseServer(String stackName, String resourceGroupName, String template, AzureClient client) {
         LOGGER.debug("Re-deploying database server {} in resource group {}", stackName, resourceGroupName);
         String parametersMapAsString = new Json(Map.of()).getValue();
-        client.createTemplateDeployment(resourceGroupName, stackName, template, parametersMapAsString);
+        try {
+            retryService.testWith2SecDelayMax5Times(() -> {
```

Comment:
do you think this is likely to get resolved in 10 sec? I am afraid that it is not, so I would rather go with a backoff type of retry like `testWith1SecDelayMax5Times` but with a higher `maxDelay` set, polling for at least 1 minute.

Reply:
I have no idea how long azure takes to realize the db is really gone. we could increase this later if required. Also it would be more than 10 sec as the request etc takes time